recipe/meta.yaml

```diff
@@ -24,12 +33,15 @@ build:
   skip: true  # [win]
   # For some reason Python 3.12 fails to resolve and gives:
   # nothing provides _python_rc needed by python-3.12.0rc3-rc3_hab00c5b_1_cpython
-  skip: true  # [cuda_compiler_version != "None" and py==312]
-  number: 6
+  skip: true  # [(cuda_compiler != "None" and py==312) or cuda_compiler_version not in ("None", "11.8", "12.0")]
```

Comment:
What's the reason for skipping 11.2?

Reply:
Our plan to kick off the 11.8 migration (#8) is to retire the CUDA 11.2 infra, because it's getting old and by utilizing CUDA minor version compatibility we can build with 11.8 and serve all 11.x users, which is what's enabled in this PR and in other feedstocks.